src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1892,7 +1892,7 @@ public async Task<JObject> CreateJObjectForValueType(MonoBinaryReader retDebugge
             else if (isBoxed && numFields == 1) {
                 return fieldValueType;
             }
-            return CreateJObject<string>(null, "object", description, false, className, $"dotnet:valuetype:{valueTypeId}", null, null, true, true, isEnum == 1);
+            return CreateJObject(valueTypes[valueTypeId].valueTypeJson, "object", description, false, className, $"dotnet:valuetype:{valueTypeId}", null, null, true, true, isEnum == 1);
```

Comment:
`valueTypeJson` here is:
```json
 [
  {
    "value": {
      "type": "number",
      "value": 1,
      "description": "1"
    },
    "writable": true,
    "isOwn": true,
    "name": "value__"
  }
]
```

The whole thing isn't needed, just extract the actual value, and use that, like `valueTypeJson[0]["value"]`.

This is kinda [misusing](https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#type-RemoteObject) the `value` field in the object that we create though. In a follow up PR, we should return a better result object for enums.